src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/DeferredContent/QuickInfoDisplayDeferredContent.cs

```diff
@@ -34,6 +36,7 @@ internal class QuickInfoDisplayDeferredContent : IDeferredQuickInfoContent
             AnonymousTypes = anonymousTypes;
             UsageText = usageText;
             ExceptionText = exceptionText;
+            CapturesText = capturesText;
```

Comment:
FYI to both @JieCarolHu and @DustinCampbell as a collision warning for other quick info things happening.